### ISSUE TITLE
More colors combinations

### ISSUE THREAD

**Issue by Andrea-Giordano** [felixblaschke/simple_animations#48]
Hi, I'd like to report a suggestion: I've been looking exactly for the effect you named "Plasma" and it looks amazing. I think I would be even greater if there was a simple way to have more than 2 colors (particles of different colors?)

**Comment by felixblaschke** [comment]
I came up with the same suggestion to myself right after I released that plasma effect. I am currently working on something big. But it takes some time to get it finished.

Here's sneak preview of a possible result: (Red and yellow plasma on blue background.)

<img width="761" alt="Screenshot 2020-12-17 at 23 33 03" src="https://user-images.githubusercontent.com/26407468/102551771-5e01b200-40c0-11eb-8045-789557e93ca7.png">
